lms/djangoapps/bulk_email/models.py

```diff
@@ -127,7 +156,7 @@ def ensure_valid_cohort(cls, cohort_name, course_id):
         if cohort_name is None:
             raise ValueError("Cannot create a CohortTarget without specifying a cohort_name.")
         try:
-            cohort = CourseUserGroup.get(name=cohort_name, course_id=course_id)
+            cohort = CourseUserGroup.objects.get(name=cohort_name, course_id=course_id)
```

Comment:
Now using this helper from `cohorts.py` https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/course_groups/cohorts.py#L286